firefox-ios/firefox-ios-tests/Tests/XCUITests/InactiveTabsTest.swift

```diff
@@ -0,0 +1,137 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import XCTest
+
+final class InactiveTabsTest: BaseTestCase {
+    override func setUp() {
+        launchArguments = [
+            LaunchArguments.PerformanceTest,
+            LaunchArguments.SkipIntro,
+            LaunchArguments.SkipWhatsNew,
+            LaunchArguments.SkipETPCoverSheet,
+            LaunchArguments.SkipContextualHints,
+            LaunchArguments.DisableAnimations
+        ]
+
+        // Load 20 tabs. 19 tabs are inactive.
+        let tabsDatabase = "tabsState20.archive"
+        launchArguments.append(LaunchArguments.LoadTabsStateArchive + tabsDatabase)
+        launchArguments.append(LaunchArguments.LoadDatabasePrefix + tabsDatabase)
+
+        super.setUp()
+
+        // Workaround: Restart app to guarantee the tabs are loaded
```

Comment:
Have you tried using the restart functions already implemented in BaseTestCase?

Reply:
I tried. Both don't work for different reasons on my computer.
* restart(): Force loading the database requires `app.terminate()` before another `app.activate()`.
* forceRestartApp(): The `setUp()` calls `setup()` here recursively instead of `super.setUp()` 🤦🏼‍♀️ .

Reply:
This test needs its own `setUp()` to set the `launchArguments` for loading the custom database.